bootstrap.sh

```diff
@@ -4,9 +4,19 @@
 git submodule init
 git submodule update
 
-# install eigen, keyutils, kerberos, sparsehash, gflags, glog
+# install eigen, keyutils, kerberos, sparsehash, gflags, glog, lz4
 cd third_party
 
+if [ ! -d "lz4-dev" ]; then
+  wget https://github.com/lz4/lz4/archive/dev.zip
+  unzip dev.zip
+fi
+cd lz4-dev
+make
+make install
```

Comment:
Right now the LDA tests are failing on travis because we don't have permissions to do 'make install' there. To fix this do:

1. Remove this make install and sudo make install.
2. Add to the .travis.yml file an "apt-get install liblz4-dev".
3. Add liblz4-dev to the cirrus/README